### ISSUE TITLE
Question to clarify 1.4.11 Non-text Contrast (AA)

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#861]
Boundaries

> This success criteria does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed. If a button with text also has a colored border, since the border does not provide the only indication there is no contrast requirement beyond the text contrast (1.4.3 Contrast (Minimum)). Note that for people with cognitive disabilities it is recommended to delineate the boundary of controls to aid in the recognition of controls and therefore the completion of activities.

Example

A button has white text on a green boundary (background), and the green button boundary is on a grey background (page). 
If the white button text against the boundary (button background) does not equal or exceed a contrast ratio of 4.5:1 this fails the SC for 1.4.3 Contrast (Minimum).

But, if the green button background does not meet the requirements for 1.4.11 Non-text Contrast, where the background is less than 3.1:1 against the adjacent grey background. Is that a fail for 1.4.11 Non-text Contrast. Or, is the white button text against the green button background a 1.4.11 Non-text Contrast failure? Does this fail 1.4.11 Non-text Contrast at all.

**Comment by alastc** [comment]
If the white text on a green background passes 1.4.3, and the boundary is not the only way of idenfiying the control then it would not be in scope of 1.4.11.

In the context of designs which may not have any background/border change for identifying buttons/links, the requirement is low. Basically, so long as it is not randomly intermingled with text then a border/background is not required. 

We just added example three to this [table in the understanding](https://w3c.github.io/wcag/understanding/non-text-contrast.html#active-user-interface-component-examples). Which I think answers your question. This previous issue #800 is also very close to your question.

Does that answer it?

**Comment by LaurenceRLewis** [comment]
@alastc It does. Thank you.
